tests/data/dash_or_underscore_0/Pipfile

```
@@ -0,0 +1,14 @@
+[[source]]
+name = "pypi"
+url = "https://pypi.org/simple"
+verify_ssl = true
+
+[dev-packages]
+gitdir = "*"
+generic-package = { editable = true, path = "." }
+
+[packages]
+xml_subsetter = "*"
```

To cover the hyphen case:

```suggestion
xml_subsetter = "*"
pytest-cov = "*"
```

---

I was considering this, though I didn't see a side effect of checking for underscores at the same time as checking for dashes. I suppose the assumption was a test for dashes would have already been implemented in another test, and if none of the other tests had failed as a result, then this would have been OK.

---

Yup that's completely valid. I imagine `pip` won't change its standard in this behavior at any time in the near future, so feel free to leave it either way 🙂